index.html

```diff
@@ -6,9 +6,9 @@
     <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
     <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
     <title>Real Dev Squad</title>
-    <link rel="stylesheet" href="./css/main.css" />
-    <link rel="stylesheet" href="./css/animate.css" />
-    <link rel="stylesheet" href="./css/navbar.css" />
+    <link rel="stylesheet" href="./css/main.css" defer />
```

Comment:
defer is used for scripts https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script